app/containers/markdown/Emoji.tsx

```diff
@@ -8,11 +7,24 @@ import { themes } from '../../constants/colors';
 
 import styles from './styles';
 
+// TODO this interface it's duplicated with 'app/containers/EmojiPicker/index.tsx'
```

Comment:
Kind of duplicated, yes, but it contains a few more props than EmojiPicker.

Reply:
yeah, I'll remove this `TODO` because we already discussed about it.